package.json

```diff
@@ -36,7 +36,7 @@
     "LICENSE",
     "README.md",
     "index.js",
-    "_locales/.gitkeep"
+    "translations/.gitkeep"
```

Comment:
@wilbertverayin Commit this folder as well. Also, add `"prepare": "grunt build"` under `scripts` so that the grunt build is always run on install https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts

Also, so that your test can be replaced with installing the package instead of building locally. The inclusion of `prepare` should allow us to use do `npm i anyTV/anytv-i18n#HD-112629-fix-i18n-config`

Reply:
I've tested this with `npm i anyTV/anytv-i18n#branch-test-install` which branches out from your branch. Try it as well on your end if it would work.

Reply:
Actually when I tried this (with normalized folders, the i18n gets run on the project level (ex. running it on content-id-service with default folder on path.resolve('translations'), will create a translations folder now under content-id-service

Reply:
@wildtron 

I have not tried that npm i <branch>. I thought it would need a build file included on the branch we want to npm i?

Reply:
@wilbertverayin I realized, you can also use `npm run build` instead in the prepare script so that it calls the build. :D

Reply:
Updated this